features/alarms/history/elastic/src/main/java/org/opennms/features/alarms/history/elastic/ElasticHealthCheck.java

```diff
@@ -54,6 +54,16 @@ public String getDescription() {
         return "Number of active alarms stored in Elasticsearch (Alarm History)";
     }
 
+    @Override
+    public String getName() {
+        return "opennms-es-alarmhistory";
```

Comment:
All the names can be made constants in HealthCheck interface so that new additions will follow the same pattern in naming.

Reply:
I think I get where you're going with this... Leme see what I can do.